src/component/stats.ts

```diff
@@ -43,3 +45,46 @@ export function recordCompleted(
     lagSinceEnqueued: Date.now() - work._creationTime,
   });
 }
+
+/**
+ * Warning: this should not be used from a mutation, as it will cause conflicts.
+ * Use this to debug or diagnose your queue length when it's backed up.
+ */
+export const queueLength = query({
```

Comment:
how about we make this into an internal query so you can call it from the dashboard but nowhere else?